src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/ColumnDefinition.cs

```diff
@@ -0,0 +1,1073 @@
+using MS.Utility;
```

Comment:
As this will be the part of source code now.

```suggestion

// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.

using MS.Utility;
```
```suggestion
using MS.Utility;
```